middle_end/flambda2/simplify/env/downwards_env.ml

```diff
@@ -47,14 +47,15 @@ type t =
     all_code : Code.t Code_id.Map.t;
     inlining_history_tracker : Inlining_history.Tracker.t;
     loopify_state : Loopify_state.t;
-    continuation_stack : Continuation.t list;
-    variables_defined_in_current_continuation : Lifted_cont_params.t;
-    variables_defined_in_current_continuation' : Lifted_cont_params.t;
     lifted : Variable.Set.t;
```

Comment:
This field provides a fast access to the set of variables defined in the current continuation (the head of `defined_variables_by_scope` below). It should be moved down in the record and documented as such.